ligo-virgo-emfollowup-tutorial.ipynb

```diff
@@ -907,6 +908,64 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## 7. Submitting observation coordinates to GraceDB\n",
+    "\n",
+    "Suppose you have performed some EM observations to follow up on a candidate GW event, and you now want to supply the coordinates of those observations to the LV-EM group.  The first step is to obtain a robotic access password from GraceDB and add it to a netrc file (see Section 2). One can then use the Python GraceDB client in order to submit a list of coordinates to GraceDB. To install the GraceDB client package:\n",
+    "\n",
+    "    $ pip install \"ligo-gracedb>=1.18.dev0\" \n",
```

Comment:
Well, the features that allow creation of EMObservation objects aren't in production yet. It could be before ER7, though, if we expect people to be trying this out.